app/experimenter/templates/experiments/list.html

```diff
@@ -27,11 +27,15 @@ <h3 class="m-0">
       {{ filter.form.get_project_display_value }}
     {% endif %}
 
-    Experiment{{ experiments|pluralize:"s" }}
+    Experiment{{ paginator.count|pluralize:"s" }}
 
     {% if filter.form.owner.value %}
       by {{ filter.form.get_owner_display_value }}
     {% endif %}
+
+    {% if is_paginated and page_obj.number > 1 %}
+      <small class="text-muted">Page {{ page_obj.number }}</small>
```

Comment:
I took the liberty to add this. I know it's a bit out of context for [the issue](https://github.com/mozilla/experimenter/issues/962) but it certainly feels and looks great. I'm happy to change it to whatever you prefer but I think it's good UI practice to indicate, in some form, that what you're looking at is page X. 

Also, omitted to say "Page 1" if you're on page 1. That's just to reduce the noise. Unless someone is clicking around on the paginator (at the bottom) they most "oftenly" don't care that you're on page 1.

Reply:
I like this, especially because with the default page size (50, I think?) the paginator is very far away from the top. I think it would be ever better as a "Page 1 of X" indicator, but I'm not sure about that.

Reply:
I'm not strongly opinionated on the best practice of that. 

At the risk of over-analyzing this; this is what Google does with page 1 and page 2:

<img width="712" alt="screen shot 2019-02-22 at 8 46 13 am" src="https://user-images.githubusercontent.com/26739/53246363-7cdabd00-367e-11e9-9706-d7985d5a2c3f.png">

<img width="700" alt="screen shot 2019-02-22 at 8 46 22 am" src="https://user-images.githubusercontent.com/26739/53246391-86642500-367e-11e9-91a7-3ed0e0906dfa.png">

Reply:
I just noticed this, it does look good!